rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Let's add the `:book and :person` to `attr_accessor` to allow users to be able to read and write them. See suggestion below:
```suggestion
  attr_accessor :date, :book, :person
```

Reply:
I have made the changes requested